internal/sign/helper_test.go

```diff
@@ -0,0 +1,174 @@
+package sign
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+var _ = Describe("GetRelevantSign", func() {
+
+	const (
+		unsignedImage = "my.registry/my/image"
+		keySecret     = "securebootkey"
+		certSecret    = "securebootcert"
+		filesToSign   = "/modules/simple-kmod.ko:/modules/simple-procfs-kmod.ko"
+	)
+
+	var (
+		ctrl *gomock.Controller
+		h    Helper
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		h = NewSignerHelper()
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
```

Comment:
I think the gomock controller does that automatically - no need to call `AfterEach` manually.